.circleci/common.sh

```diff
@@ -196,6 +196,13 @@ function run_torch_xla_cpp_tests() {
   popd
 }
 
+function run_torch_xla_benchmark_tests() {
```

Comment:
I did that at first. However @JackCaoG asked me to move these tests to their own run_tests.sh file, since they're not Python tests per se (as the ones in xla/test/run_tests.sh). That's why I added a new xla/test/benchmarks/run_tests.sh file. (Note that we also have xla/test/cpp/run_tests.sh; those tests aren't Python tests either.)